### PR TITLE
FOUR-9229 Review email-connect, to avoid the launch of a docker instance to send an simple email

### PR DESCRIPTION
**Pull request by caleeli**
## Issue & Reproduction Steps
Bring up a docker ssr to render a html template is a bit overkill, so as a developer I want to use a simple php renderer with mustache and date time format support.

### Expected behavior: 
Add HtmlRenderer class with mustache and date format

## Solution
- Implement a HtmlRenderer class with mustache and date format support.

## How to Test
- Create a process with date and datetime fields and with connector email (e.g. attached process)
- Run the process
- Check the variables are replaced in the email body and the date and time are formatted as the system configuration.

[Test_EmailConnector.zip](https://github.com/ProcessMaker/connector-send-email/files/12021230/Test_EmailConnector.zip)

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-9229

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

### CODE REVIEW

File: ProcessMaker/Nayra/Managers/WorkflowManagerRabbitMq.php [hunk 1]
```diff
@@ -302,6 +320,7 @@ public function runServiceTask(ServiceTaskInterface $serviceTask, TokenInterface
             $token->setProperty('error', $error);
 
             // Log message errors
+            error_log('Service task failed: ' . $implementation . ' - ' . $exception->getMessage());
```
Comment (julceslauhub) [inline ProcessMaker/Nayra/Managers/WorkflowManagerRabbitMq.php:323]: please remove this line, in the next line is logged the same error...

File: ProcessMaker/Nayra/Managers/WorkflowManagerRabbitMq.php [hunk 2]
```diff
@@ -10,13 +10,15 @@
 use ProcessMaker\Facades\MessageBrokerService;
 use ProcessMaker\Facades\WorkflowManager;
 use ProcessMaker\GenerateAccessToken;
+use ProcessMaker\Jobs\RunNayraServiceTask;
 use ProcessMaker\Managers\DataManager;
 use ProcessMaker\Managers\SignalManager;
 use ProcessMaker\Models\EnvironmentVariable;
-use ProcessMaker\Models\Process as Definitions;
 use ProcessMaker\Models\Process;
+use ProcessMaker\Models\Process as Definitions;
```
Comment (julceslauhub) [inline ProcessMaker/Nayra/Managers/WorkflowManagerRabbitMq.php:18]: this reference is not used in this file...

File: ProcessMaker/Nayra/Managers/WorkflowManagerRabbitMq.php (hunk 1 above)
Comment (caleeli) [reply ProcessMaker/Nayra/Managers/WorkflowManagerRabbitMq.php:323]: Removed

File: ProcessMaker/Nayra/Managers/WorkflowManagerRabbitMq.php (hunk 2 above)
Comment (caleeli) [reply ProcessMaker/Nayra/Managers/WorkflowManagerRabbitMq.php:17]: Definitions is used as alias of Process. I removed the `use ProcessMaker\Models\Process;` and use `Definitions` to keep the logic of the class

**Review by julceslauhub — APPROVED** [review]
Approved

**Pull request merged**